cfme/tests/cloud_insights/test_cloud_insights_manual.py

```diff
@@ -1,112 +0,0 @@
-import pytest
-
-from cfme import test_requirements
-
-pytestmark = [
-    pytest.mark.tier(0),
-    test_requirements.access,
-    pytest.mark.manual
-]
-
-
-def test_red_hat_cloud_services_overview():
```

Comment:
Q: are these test already automated or no more valid?

Reply:
@digitronik the CFME+insights integration feature was removed completely